--- gitweb.cgi.orig	2012-11-24 09:21:22.000000000 +0000
+++ gitweb.cgi	2013-09-01 12:24:08.000000000 +0000
@@ -4016,7 +4016,8 @@
 	my %opts = @_;
 
 	my $title = get_page_title();
-	my $content_type = get_content_type_html();
+	#my $content_type = get_content_type_html();
+	my $content_type = "text/html";
 	print $cgi->header(-type=>$content_type, -charset => 'utf-8',
 	                   -status=> $status, -expires => $expires)
 		unless ($opts{'-no_http_header'});
